README.md

```diff
@@ -30,19 +52,24 @@ SETX FUNCTIONAL_TEST_EMAIL "valid email address"
 SETX EMAIL_TEMPLATE_ID "valid email_template_id"
 SETX SMS_TEMPLATE_ID "valid sms_template_id"
 SETX LETTER_TEMPLATE_ID "valid letter_template_id"
+SETX SMS_SENDER_ID "valid sms_sender_id - to test sending to a receiving number, so needs to be a real number"
+SETX API_SENDING_KEY "API_whitelist_key for sending an SMS to a receiving number"
+SETX INBOUND_SMS_QUERY_KEY "API_test_key to get received text messages"
 ```
 </details>
 
 ### [Visual Studio](https://www.visualstudio.com/vs/visual-studio-mac/) (Mac OS)
 
-In order to get the .Net client running in Visual Studio the target `.Net Framework` needs to be set to `4.5.2` and the application needs to be run from the terminal
+In order to get the .Net client running in Visual Studio the target `.Net Framework` needs to be set to `4.5.2` and the application needs to be run from the terminal.
```

Comment:
This is 4.6.2 with the latest master. However can we avoid version numbers here to avoid updating in future?